pkg/util/mon/bytes_usage.go

```diff
@@ -264,11 +265,20 @@ type BytesMonitor struct {
 	settings *cluster.Settings
 }
 
-// enableMonitorTreeTracking indicates whether tracking of all children of a
-// BytesMonitor (which is what powers TraverseTree) is enabled.
-var enableMonitorTreeTracking = envutil.EnvOrDefaultBool(
+// enableMonitorTreeTrackingEnvVar indicates whether tracking of all children of
+// a BytesMonitor (which is what powers TraverseTree) is enabled.
+var enableMonitorTreeTrackingEnvVar = envutil.EnvOrDefaultBool(
 	"COCKROACH_ENABLE_MONITOR_TREE", true)
 
+// enableMonitorTreeTrackingSetting indicates whether tracking of all children
+// of a BytesMonitor (which is what powers TraverseTree) is enabled.
```

Comment:
nit: You could mention how both the env var and setting need to be enabled to enable monitor tracking, in a future PR. No need to change the backport, though.